webapp/src/main/java/com/box/l10n/mojito/rest/repository/RepositoryWS.java

```diff
@@ -72,29 +73,17 @@ public Repository getRepositoryById(@PathVariable Long repositoryId)
     return repository;
   }
 
-  /**
-   * Gets all undeleted repositories with @{link View.RepositorySummary}
-   *
-   * @return List of {@link Repository}s
-   */
-  @JsonView(View.RepositorySummary.class)
-  @RequestMapping(value = "/api/repositories", method = RequestMethod.GET)
-  @ResponseStatus(HttpStatus.OK)
-  public List<Repository> getRepositories() {
-    return repositoryService.findRepositoriesIsNotDeletedOrderByName(null);
-  }
-
   /**
    * Gets repository matching the given name
    *
    * @param repositoryName To filer on the name. Can be {@code null}
    * @return List of {@link Repository}s
    */
   @JsonView(View.Repository.class)
-  @RequestMapping(value = "/api/repositories", params = "name", method = RequestMethod.GET)
+  @RequestMapping(value = "/api/repositories", method = RequestMethod.GET)
   @ResponseStatus(HttpStatus.OK)
   public List<Repository> getRepositories(
```

Comment:
Two methods with the same path and HTTP method (GET) are not supported by Open API Specification 3.0